### ISSUE TITLE
Using bull-board with bullmq and typescript

### ISSUE THREAD

**Issue by ankitjena** [felixmosh/bull-board#141]
## Description
I was trying to setup bull-board for one of my codebases. It is written in typescript and I use `bullmq`. I saw a PR which had the description that it added support for `bullmq`. The first error I am getting it is that 
```
Type 'Queue<any>' is missing the following properties from type 'Queue<any>': isReady, process, empty, nextRepeatableJob, and 7 more.
```
I can make it work by forcing to `never` type on the `setQueues` method, but on opening the UI, it says no stats to display, even if there are(I checked with `bull-repl`

## Question
How can I make it work with `typescript` and `bullmq`

Thanks in advance

**Comment by paeolo** [comment]
@ankitjena You are right a PR has landed for this, be sure to use bull-board 1.0.0-alpha.X, which is not the case if you install it with a classic
```command
yarn add bull-board
```

**Comment by felixmosh** [comment]
Closing the issue, @ankitjena feel free to reopen if still relevant
